README.md

```diff
@@ -16,107 +16,25 @@ Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 
 ## Usage example
 
-The example below shows 2 pipelines. `UDPDemo.Send` downloads an example file over HTTP and
-sends it over UDP socket via localhost:
+The `example/` shows 2 pipelines. 
 
-```elixir
-defmodule UDPDemo.Send do
-  use Membrane.Pipeline
-
-  alias Membrane.{Hackney, UDP}
-  alias Membrane.Pipeline.Spec
-
-  def handle_init(_) do
-    children = [
-      source: %Hackney.Source{
-        location: "https://membraneframework.github.io/static/video-samples/test-video.h264"
-      },
-      udp: %UDP.Sink{
-        destination_address: {127, 0, 0, 1},
-        destination_port_no: 5001,
-        local_address: {127, 0, 0, 1}
-      }
-    ]
-
-    links = %{
-      {:source, :output} => {:udp, :input}
-    }
-
-    spec = %Spec{children: children, links: links}
-    {[spec: spec], %{}}
-  end
-end
+The `UDPDemo.Send` downloads an example file over HTTP and sends it over UDP socket via localhost:
+```bash
+$ elixir examples/sink.exs
 ```
 
-The `UDPDemo.Receive` retrieves packets from UDP socket and
-saves the data to the `/tmp/udp-recv.h264` file.
+The `UDPDemo.Receive` retrieves packets from UDP socket and saves the data to the `/tmp/udp-recv.mp4` file.
+```bash
+$ elixir examples/source.exs
+```
```

Comment:
- I'd name them `send.exs` and `receive.exs`
- Let's mention that the receiving script should be started before the sending script